src/prog_models/prognostics_model.py

```diff
@@ -17,6 +17,7 @@
 from .utils.containers import DictLikeMatrixWrapper
 from .utils.parameters import PrognosticsModelParameters
 from .utils.serialization import *
+from .utils.size import getsizeof
```

Comment:
I thought it was decided to not use relative path imports anymore.

Reply:
It was, good catch!